### DISCUSSION TITLE
unable to load image

### DISCUSSION THREAD

**Discussion by peterlymo** [CaiJimmy/hugo-theme-stack#794] [Q&A]
i was configuring this themes but i cant make image or featured image to display, what am missing

---
title: english Test
description: oops! this is
date: 2020-09-09
image: ctf.jpg
categories:
    - Test
draft: true
---

![Image 1](ctf.jpg)

no image display

my image under hello folder with hello.md

**Comment by peterlymo** [comment]
```---
title: english Test
description: oops! this is
date: 2020-09-09
image: ctf.jpg
categories:
    - Test
draft: true
---


![Image 1](ctf.jpg)
```

**Comment by rmdhnreza** [comment]
> my image under hello folder with hello.md

Try changing `hello.md` to `index.md`, so the structure looks like this

```md
content/
└── post
    └── hello
        ├── ctf.jpg
        └── index.md
```

**Reply by cohhi** [reply]
Thank you for your answer. My problem has been resolved 😄
